suite-native/atoms/src/Headered.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from 'react';
+
+import { G } from '@mobily/ts-belt';
+
+import { VStack } from './Stack';
+import { Text } from './Text';
+import { Box } from './Box';
+
+type HeaderedProps = {
+    header: ReactNode;
+    children: ReactNode;
+};
+
+type HeaderContainerProps = { children: ReactNode };
+
+const HeaderContainer = ({ children }: HeaderContainerProps) => (
+    <Box paddingHorizontal="medium">
+        {G.isString(children) ? <Text>{children}</Text> : children}
+    </Box>
+);
+
+/**
+ * Adds a header to child component with app-wide padding styles.
+ */
```

Comment:
nit: no need for this kind of comment. Shows bad naming if you need comment like this. `Headered` is not saying anythin